workshops/portfolio/README.md

```diff
@@ -11,21 +11,70 @@ Your final design looks like this:
 
 > ![](img/final_1.png)
 
-Here is a link to a [live demo](http://output.jsbin.com/fugoki/2)
+Here is a link to a
+<a href="http://output.jsbin.com/fugoki/2" target="_blank">live demo</a>
```

Comment:
I don't think we should use `a` tags when we really don't have to, for the sake of Markdown. I'd revert this change.